### PR TITLE
Media queries

### PR DESCRIPTION
**Pull request by collinkallery**
# PR Description
Please include a summary of the change:
- Three different breakpoints for desktop, tablet, and phone. 
- Added various HTML classes to work with breakpoints. 

Commit Emojis:
:construction: HTML modifications/updates.
:art: CSS modifications/updates.
:crystal_ball: JS modifications/updates.
:lipstick: hotfix any code.

If it is a new feature, describe the feature and the iteration of the project spec it addresses.
- This addresses the platform compatibility section of the project spec. Moving forward, we need to REMOVE any common properties from rules within media queries that are not changing across breakpoints, and move them up into the main declarations. 
- We also need to create the mixins and variables and implement them across our SCSS. 

Note any questions or struggles as well.
Currently wondering if it makes sense to remove common properties from breakpoints and declare them above as well but we will cross that bridge when we get to it. 

## Type of change
Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:
- [x] My code follows the Turing style guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: Refactor Tractor.mindnode/style.mindnodestyle/contents.xml [hunk 1]
```diff
@@ -0,0 +1,274 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Karlfunhouse) [inline Refactor Tractor.mindnode/style.mindnodestyle/contents.xml:1]: DELETE THIS

File: Refactor Tractor.mindnode/style.mindnodestyle/metadata.plist [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Karlfunhouse) [inline Refactor Tractor.mindnode/style.mindnodestyle/metadata.plist:1]: delete this too

File: src/css/styles.scss [hunk 3]
```diff
@@ -423,3 +278,283 @@ main {
 .yellow-text {
   color: #F8BA00;
 }
+/* MEDIA QUERIES */
```
Comment (Karlfunhouse) [inline src/css/styles.scss:281]: Desktop Sizing.

File: src/css/styles.scss [hunk 4]
```diff
@@ -423,3 +278,283 @@ main {
...
+    border: none;
+    height: 8vw;
+    margin-left: 1vw;
+    width: 8vw;
+  }
+  .hide {
+    display: none;
+  }
+}
+@media screen and (max-width: 414px) {
```
Comment (Karlfunhouse) [inline src/css/styles.scss:547]: Phone Screen Size

File: src/css/styles.scss [hunk 5]
```diff
@@ -423,3 +278,283 @@ main {
...
+    border: none;
+    height: 3.4vw;
+    margin-left: 1vw;
+    width: 3.4vw;
+  }
+  .hide {
+    display: none;
+  }
+}
+@media screen and (max-width: 768px) {
```
Comment (Karlfunhouse) [inline src/css/styles.scss:418]: Tablet screen size

**Comment by Melizzo** [comment]
Very impressive Collie, looks amazing.

**Pull request merged**